src/Native/Console/NativeCom.js

```diff
@@ -58,23 +65,25 @@ Elm.Native.Console.NativeCom.make = function(localRuntime) {
     }
 
     var doRequest = function(request) {
-        switch(request.ctor) {
-            case 'Put':
-                process.stdout.write(request._0);
-                break;
-            case 'Get':
-                stdin.resume();
-                break;
-            case 'Exit':
-                process.exit(request._0);
-                break;
-            case 'WriteFile':
-                fs.writeFileSync(request._0.file, request._0.content);
-                break;
-            case 'Init':
-                // trigger the initial IO requests
-                sendResponseString(null);
-                break;
+        if (inNode) {
+            switch(request.ctor) {
+                case 'Put':
+                    process.stdout.write(request._0);
+                    break;
+                case 'Get':
+                    stdin.resume();
+                    break;
+                case 'Exit':
+                    process.exit(request._0);
+                    break;
+                case 'WriteFile':
+                    fs.writeFileSync(request._0.file, request._0.content);
+                    break;
+                case 'Init':
+                    // trigger the initial IO requests
+                    sendResponseString(null);
+                    break;
+            }
```

Comment:
@deadfoxygrandpa my concern is here. This fails silently if you are trying to write something to the console. It does not need to be wrapped with `isNode` because unlike at the top of the file, this part doesn't run by default.